cmd/gapit/flags.go

```diff
@@ -409,4 +409,13 @@ type (
 		DeviceFlags
 		Gapis GapisFlags
 	}
+
+	PerfettoFlags struct {
+		Mode				string	`help:"Run mode: {metrics|interactive}. Default: metrics."`
+		Trace				string	`help:"Trace file."`
```

Comment:
For most other gapit commands, the trace is always the last argument (and is not in the named list)

Reply:
Done.